ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,278 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  /**
+   * Common assertions for Add terms, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
+   * @param {Object} component - Root DOM object in react class
+   * @return {Object} devItems - DOM object of class 'panel-body'
+   * */
+  function  assertAddTermCommon(assert, component) {
+    // check term title
+    var listOfLinks = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        component,
+        'a'
+    );
+    assert.equal(listOfLinks.length, 2);
+    var linkHeader = listOfLinks[0].getDOMNode();
+    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    //test items
+    var devItems = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'panel-body'
+    );
+    var itemList = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        devItems,
+        'li'
+    );
+    assert.equal(itemList.length, 2);
+    return devItems;
+  }
+  /**
+   * Common assertions for Add Vocabulary, use in term and taxonomy
+   * components tests
+   * @param {Object} assert - For assertions
+   * @param {Object} component - Root DOM object in react class
+   * @return {Object} devItems - DOM object of class 'form-horizontal'
+   * */
+  function assertAddVocabularyCommon(assert, component) {
+    var formNode = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'form-horizontal'
+      );
+    assert.ok(formNode);
+    //test form submission
+    var inputNodes = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        formNode,
+        'input'
+      );
+    assert.equal(inputNodes.length, 12);
+    var inputVocabularyName = inputNodes[0];
+    var inputVocabularyDesc =  inputNodes[1];
+    var radioManaged =  inputNodes[10];
+    var radioTagStyle = inputNodes[11];
+
+    React.addons.TestUtils.Simulate.change(
+      inputVocabularyName, {target: {value: "TestA"}}
+    );
+    React.addons.TestUtils.Simulate.change(
+      inputVocabularyDesc, {target: {value: "TestA"}}
+    );
+    radioManaged.checked = true;
```

Comment:
You are setting the checked attribute to true here but in the next line you are telling React to set it to false. It seems like you can just remove this line since we're only asserting React's state here